test/powershell/Language/Scripting/CheckRestrictedlanguage.Tests.ps1

```diff
@@ -0,0 +1,112 @@
+﻿#  <Test>
+#    <summary>Test restricted language check method on scriptblocks</summary>
+#  </Test>
+Describe "Test restricted language check method on scriptblocks" -Tags "CI" {
+        BeforeAll {
+            set-strictmode -v 2
+            function list {
+
+            $l = new-object system.collections.generic.list[string]
```

Comment:
>w-object system.collections.generic.list[string] [](start = 19, length = 48)

i think the form we're suggesting now is:
`[System.Collections.Generic.List[String]]::new()`